packages/plugins/RSS3/src/SNSAdaptor/pages/DonationsPage.tsx

```diff
@@ -48,31 +39,43 @@ const useStyles = makeStyles()((theme) => ({
 }))
 
 export interface DonationPageProps {
-    donations?: GeneralAsset[]
+    donations?: RSS3BaseAPI.Donation[]
     loading?: boolean
-    addressLabel: string
+    address: SocialAddress<NetworkPluginID>
 }
 
-export function DonationPage({ donations = [], loading, addressLabel }: DonationPageProps) {
+export function DonationPage({ donations = [], loading, address }: DonationPageProps) {
     const { classes } = useStyles()
     const t = useI18N()
 
+    const [selectedDonation, setSelectedDonation] = useState<RSS3BaseAPI.Donation | undefined>()
+
     if (loading || !donations.length) {
-        return <StatusBox loading={loading} empty={!donations.length} />
+        return <StatusBox loading={loading} collection="Donation" empty={!donations.length} />
```

Comment:
Refactor to use an enum value.

```suggestion
        return <StatusBox loading={loading} collection={CollectionType.Donation} empty={!donations.length} />
```